frontend/public/components/resource-dropdown.tsx

```diff
@@ -89,8 +90,19 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = props => {
     .toJS() as {[s: string]: JSX.Element};
 
   const selectedKey = allItems[selected] ? selected : kindForReference(selected);
-
-  return <Dropdown className={classNames('co-type-selector', className)} items={allItems} title={allItems[selectedKey]} onChange={onChange} selectedKey={selectedKey} />;
+  const autocompleteFilter = (text, item) => {
+    const searchString = item.props.model ? item.props.model.abbr + item.props.model.kind : null;
+    return fuzzy(_.toLower(text), _.toLower(searchString));
+  };
+
+  return <Dropdown
```

Comment:
would you guys consider the [type ahead component](https://rawgit.com/patternfly/patternfly-react/gh-pages/index.html?selectedKind=patternfly-react%2FForms%20and%20Controls%2FTypeAhead%20Select&selectedStory=Menu%20Alignment&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs) from patternfly-react? 

thanks